admin/class-meta-columns.php

```diff
@@ -182,30 +183,23 @@ public function column_sort( $columns ) {
 	/**
 	 * Hides the SEO title, meta description and focus keyword columns if the user hasn't chosen which columns to hide.
 	 *
-	 * @param array|false $result The hidden columns.
-	 * @param string      $option The option name used to set which columns should be hidden.
-	 * @param WP_User     $user   The User.
+	 * @param array  $hidden The hidden columns.
+	 * @param object $screen The current screen.
 	 *
-	 * @return array      $result Array containing the columns to hide.
+	 * @return array Array containing the columns to hide.
 	 */
-	public function column_hidden( $result, $option, $user ) {
-		global $wpdb;
-
-		if ( $user->has_prop( $wpdb->get_blog_prefix() . $option ) || $user->has_prop( $option ) ) {
-			return $result;
-		}
-
-		if ( ! is_array( $result ) ) {
-			$result = [];
+	public function column_hidden( $hidden, $screen ) {
```

Comment:
This method does not use the `$screen` parameter, so this can be removed from the list of parameters.